client/logical_port_client.go

```diff
@@ -0,0 +1,11 @@
+package client
```

Comment:
how about renaming this file into `types.go`, so that we can simply include other interfaces in the same file for ease of readibility

Reply:
ack, moved client interface to `types.go`